contribute/developer-guide.md

```diff
@@ -77,10 +77,26 @@ yarn start
 ```
 
 This command will generate sass theme files, build all external plugins, then build the frontend assets.
-Once `yarn start` has built the assets, it will continue to do so whenever any of the files change. This means you don't have to manually build the assets every time you change the code.
+Once `yarn start` has built the assets, it will continue to do so whenever any of the core Grafana application files change. This means you don't have to manually build the assets every time you change the code.
 
 > Troubleshooting: if your first build works, but after pulling updates you see unexpected errors in the "Type-checking in progress..." stage, these can be caused by the [tsbuildinfo cache supporting incremental builds](https://www.typescriptlang.org/tsconfig#incremental). You can `rm tsconfig.tsbuildinfo` and re-try.
 
+#### External plugins
+
+If you are looking to contribute to any of the external plugins found within the `public/app/plugins` directory they require running additional commands to watch and rebuild them.
+
+To build and watch all external plugins you can run the following command. Note this can be quite resource intensive as it will start separate build processes for each external plugin.
```

Comment:
```suggestion
To build and watch all plugins you can run the following command. Note this can be quite resource intensive as it will start separate build processes for each external plugin.
```